pypeit/spectrographs/jwst_nircam.py

```diff
@@ -0,0 +1,171 @@
+"""
+Module for MMT/Blue Channel specific methods.
+
+.. include:: ../include/links.rst
+"""
+import glob
+import numpy as np
+from astropy.io import fits
+from astropy.time import Time
+
+from pypeit import msgs
+from pypeit import telescopes
+from pypeit.core import framematch
+from pypeit.par import pypeitpar
+from pypeit.spectrographs import spectrograph
+from pypeit.core import parse
+from pypeit.images import detector_container
+from IPython import embed
+
+class JWSTNIRCamSpectrograph(spectrograph.Spectrograph):
+    """
+    Child to handle MMT/Blue Channel specific code
```

Comment:
I would be good to change this description

Reply:
Fixed.